src/web/components/MostViewed/MostViewedRight/MostViewedRightItem.tsx

```diff
@@ -98,18 +91,27 @@ export const MostViewedRightItem = ({ trail }: Props) => {
                         />
                     </div>
                     <div className={headlineWrapperStyles}>
-                        <SmallHeadline
-                            headlineString={trail.linkText}
-                            pillar={trail.pillar}
-                            size="tiny"
-                            showUnderline={isHovered}
-                            {...itemProps}
-                            link={{
-                                to: trail.url,
-                                visitedColour: palette.neutral[46],
-                                preventFocus: true,
-                            }}
-                        />
+                        {trail.isLiveBlog ? (
+                            <SmallHeadline
```

Comment:
Do we associate 'isLiveBlog' with always show pulsing dot? Should that be the case, i.e are they intrinsically linked so that it shouldn't need setting, we do it by `designType` maybe?

Reply:
> Do we associate 'isLiveBlog' with always show pulsing dot?

It's context dependant. So for this component the pulsing dot is never shown but in other places it's (I believe) to if the live blog is still running or if it's dead.

There's an argument for a new design type of 'Dead' which would be semantically accurate but a bit morbid.